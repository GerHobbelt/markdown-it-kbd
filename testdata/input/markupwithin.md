# Test

We can do markup within tags: [[*i*]] [[`foo`]].
