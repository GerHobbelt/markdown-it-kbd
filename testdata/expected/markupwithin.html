<h1>Test</h1>
<p>We can do markup within tags: <kbd><em>i</em></kbd> <kbd><code>foo</code></kbd>.</p>
